yangcatalog-ui/src/app/app.component.ts

```diff
@@ -2,20 +2,36 @@ import { Component, OnInit } from '@angular/core';
 import { Title } from '@angular/platform-browser';
 import { MatomoTracker } from '@ngx-matomo/tracker';
 import { TitleService } from './shared/title/title.service';
-
+import { ToastrService } from 'ngx-toastr';
+import notification_json from './shared/notification.json';
```

Comment:
Instead of importing it from the JSON file create service file `app.service.ts` where you will fetch the content of the file from http://<BASE_URL>/notifications/notification.json